api/test/wfl/system/v1_endpoint_test.clj

```diff
@@ -27,14 +28,21 @@
 (defn- verify-succeeded-workload [workload]
   (run! verify-succeeded-workflow (:workflows workload)))
 
+(defn ^:private verify-missing-internals [workload]
```

Comment:
**optional** nit: maybe call it `verify-striped-internals` or `verify-excluded-internals` as `missing` makes me feel they should be there to some extent. (I'm bad at naming so this is totally optional)

Reply:
sounds good!